nncf/quantization/quantize_model.py

```diff
@@ -275,8 +275,9 @@ def compress_weights(
     ignored_scope: Optional[IgnoredScope] = None,
     all_layers: Optional[bool] = None,
     dataset: Optional[Dataset] = None,
-    subset_size: Optional[int] = 128,
     sensitivity_metric: Optional[SensitivityMetric] = None,
+    *,
```

Comment:
+ @eaidova to comment on the impact for llm.bench.

Reply:
no impact, we use only keywords arguments for all weighs compression parameters